packages/terra-theme-context/tests/jest/__snapshots__/ThemeContextProvider.test.jsx.snap

```diff
@@ -0,0 +1,25 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`ThemeContextProvider Snapshots should render with custom props 1`] = `
+<ContextProvider
+  value={
+    Object {
+      "className": "test-theme",
+    }
+  }
+>
+  <div />
+</ContextProvider>
+`;
+
+exports[`ThemeContextProvider Snapshots should render with minimal props 1`] = `
+<ContextProvider
+  value={
+    Object {
+      "className": undefined,
```

Comment:
Should the default be `''` to work correctly with the default theme or is undefined okay?

Reply:
Either undefined or empty string work the same.